examples/cisco/demo_itd.pp

```diff
@@ -102,4 +128,63 @@
     probe_type       => default,
     weight           => 1,
   }
+
+  cisco_command_config { 'ial':
+    command => 'ip access-list ial'
```

Comment:
Instead of `cisco_command_config` can you use existing providers for this dependent config?